### ISSUE TITLE
TextField.addShortcutListener

### ISSUE THREAD

**Issue by hrj** [henrikerola/scaladin#4]
It looks like v2.0 of Scaladin is using wrapped widgets instead of extending from original Vaadin widgets as in v1.0

Consequently, some functions that I was using are missing. For example, I could do :

```
new TextField {
  addShortcutListener( blah blah)
}
```

But I have to now use the wrapped object:

```
new TextField {
  p.addShortcutListener( blah blah)
}
```

Is this correct?

Also, is it planned to have a companion object for `TextField` and other widgets, just like for `Button`?

**Comment by ripla** [comment]
The wrapper objects should support everything that the original Java Vaadin API supports, with a Scala twist of course. The missing shortcut listener on the `TextField` should be included in the upcoming 2.0 release.

Companion objects are planned, but they will probably be provided only for the most common use-cases.

**Comment by ripla** [comment]
Should be fixed with the 2.0 release.
